lib/traject/config/folio_config.rb

```diff
@@ -197,6 +197,29 @@ def holdings(record, context)
   end
 end
 
+# We update item_display once we have crez info
+to_field 'item_display_struct' do |record, _accumulator, context|
+  next if record.courses.empty?
+
+  context.output_hash['item_display_struct'].map! do |item_display_str|
```

Comment:
Do we need to wait?  https://github.com/sul-dlss/searchworks_traject_indexer/pull/1029 opened up a seam for you to set this up when we're creating the holding.

Reply:
Wish I'd known that when I picked up this ticket! I'll try it that way.

Reply:
Updated